src/main/javascript/crypto/src/Authenticator.ts

```diff
@@ -117,7 +116,7 @@ export class Authenticator {
      *  - Since this token depends on identifier attestation, continue to open iframe to attestation.id who needs to provide the proof
      */
     getIdentifierAttestation() {
-        console.log('getIdentifierAttestation. create iframe with ' + this.attestationOrigin);
+        logger(3,'getIdentifierAttestation. create iframe with ' + this.attestationOrigin);
```

Comment:
Maybe it would make sense to replace the level with an enum reflecting the level? But I understand that it is a lot of changes to make for something cosmetic